examples/demo-rollup/README.md

```diff
@@ -23,30 +21,15 @@ This is a demo full node running a simple Sovereign SDK rollup on [Celestia](htt
   - [Start the Rollup Full Node](#start-the-rollup-full-node)
   - [Sanity Check: Creating a Token](#sanity-check-creating-a-token)
   - [How to Submit Transactions](#how-to-submit-transactions)
+  - [How to Submit Transactions](#how-to-submit-transactions-1)
     - [1. Build `sov-cli`](#1-build-sov-cli)
     - [2. Generate the Transaction](#2-generate-the-transaction)
-    - [3. Submit the Transaction(s)](#3-submit-the-transactions)
-    - [4. Verify the Token Supply](#4-verify-the-token-supply)
-  - [Makefile](#makefile)
-  - [Remote setup](#remote-setup)
-- [How to Customize This Example](#how-to-customize-this-example)
-  - [1. Initialize the DA Service](#1-initialize-the-da-service)
-  - [2. Run the Main Loop](#2-run-the-main-loop)
-- [Disclaimer](#disclaimer)
-- [Interacting with your Node via RPC](#interacting-with-your-node-via-rpc)
```

Comment:
Is there a reason we removed the "Interacting with your Node via RPC" section? Happy to be convinced otherwise, but that section seems more appropriate to be in the main README.md

Reply:
I agree the main readme is a better place. I have just updated this part.